ui/components/app/confirmations/alerts/multiple-alert-modal/multiple-alert-modal.tsx

```diff
@@ -101,7 +102,8 @@ function PageNumber({
     <Text
       variant={TextVariant.bodySm}
       color={TextColor.textAlternative}
-      marginInline={2}
+      marginInline={1}
+      style={{ whiteSpace: 'nowrap' }}
```

Comment:
Noting that this <Text> component moves the Alert icon off center once another digit is added. There are ways to style this so that this does not happen e.g. using [this solution](https://stackoverflow.com/questions/32378953/keep-the-middle-item-centered-when-side-items-have-different-widths) except this one would require updates to the <ModalHeader> which I'm not sure we want to do right now


![CleanShot 2024-05-22 at 22 36 26@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/74cb770e-f8d4-48c8-81a2-8b69a14562e4)

![CleanShot 2024-05-22 at 22 35 13@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/5ac604ca-0a12-4a1b-bdec-70f07775bf7f)

![CleanShot 2024-05-22 at 22 41 21@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/58311121-17ac-4b48-bb75-83662f00bbf2)

Reply:
Now, we won't have issues aligning the icon with the icon and text on the `AlertHeader` component outside the  `ModalHeader`.